src/task_list/moduls.py

```diff
@@ -0,0 +1,146 @@
+import json
+import pathlib
+
+
+class DataJson:
+    'Класс для работы с файлом'
+
+    def __init__(self):
+        self.file = pathlib.Path('task_list/src/config/data.json').resolve()
+
+
+    def read_data_json(self):    # Чтение файла с задачами
+        with open(self.file, 'r', encoding='utf-8') as file:
+            tasks = json.load(file)
```

Comment:
nit: соре, я душный, но прикинь, у тебя файл на 250 гигов, целый отдел задачи записывает)
и ты все это грузишь в оперативку, жуть)

прочитай про yield


если пофиг, то
почему сразу return не сделать ? 

```suggestion
            return json.load(file)
```
```suggestion
            tasks = json.load(file)
```
```suggestion
            tasks = json.load(file)
```